x-pack/plugins/observability_solution/infra/public/alerting/log_threshold/components/alert_details_app_section/components/threhsold_chart/create_lens_definition.ts

```diff
@@ -78,7 +78,7 @@ function createBaseLensDefinition<D extends {}>(
             yConfig: [
               {
                 forAccessor: '607b2253-ed20-4f0a-bf62-07a1f846cca4',
-                color: '#6092c0',
+                color: euiTheme.colors.vis.euiColorVis2,
```

Comment:
Fyi, this causes a change in color for Amsterdam (blue to pink) 
If that's currently not wanted we'd need to add a condition here to use `euiColorVis1` or `euiColorVis2`.
@bryce-b Maybe you could confirm with the responsible designer?